src/internal/net/rest/api/UserApi.js

```diff
@@ -431,4 +432,41 @@ export default class UserApi {
       userID: profile.id
     })
   }
+
+  async mergeAccounts(payload) {
+
+        try {
+
+            // Begin merge
+            await this.client.request('POST', '/v1/user/merge_accounts', payload, true)
+
+            // Wait until the specified token has been added to the account
+            let timeout = Date.now() + 30000
+            while (true) {
+
+                // Get current tokens
+                let tokens = await this.getUserTokens()
+                if (tokens.find(t => t.properties.token.toLowerCase().trim() == payload.token.toLowerCase().trim())) {
+
+                    // Found, account merging is probably finished now
+                    break
+
+                }
+
+                // Not found, check if timeout is exceeded
+                if (Date.now() > timeout)
+                    throw new Error('Could not find merged token. Please refresh the page and try again.')
+
+                // Wait a bit
+                await new Promise(c => setTimeout(c, 2000))
+
+            }
+            
+        } catch (err) {
+            
+            throw new Error(err)
```

Comment:
This try/catch block does nothing at all, it's safe to remove it... In fact it's better to remove it rather than wrapping an Error inside another Error